packages/editor/src/components/block-vertical-alignment-toolbar/test/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BlockVerticalAlignmentToolbar } from '../';
+
+describe( 'BlockVerticalAlignmentToolbar', () => {
+	const alignment = 'top';
+	const onChange = jest.fn();
+
+	const wrapper = shallow( <BlockVerticalAlignmentToolbar value={ alignment } onChange={ onChange } /> );
+
+	const controls = wrapper.props().controls;
+
+	afterEach( () => {
+		onChange.mockClear();
+	} );
+
+	it( 'should match snapshot', () => {
+		expect( wrapper ).toMatchSnapshot();
+	} );
+
+	it( 'should call onChange with undefined, when the control is already active', () => {
+		const activeControl = controls.find( ( { _id } ) => _id === alignment );
```

Comment:
So this is why `_id` exists. I must admit it's very confusing and something we try to avoid at all costs. If something is hard to test, it probably should be never unit tested. We no longer maintain code coverage stats to not put expectations that everything should be unit tested.

If you think we should keep those tests, you can also use the order of controls to pick a proper one. It should be equally fragile in case those controls ever change.

Reply:
Agreed. I've fixed this here https://github.com/WordPress/gutenberg/pull/13899/commits/e00a520a735306e57a94d191faf8cf71d02125c7